Thu Nov 16 16:58:20 CET 2006  hanwen@lilypond.org
  * import os.environ into sysconfig if CROSS_COMPILING is "yes"
diff -rN -u old-Python-2.5.cross/setup.py new-Python-2.5.cross/setup.py
--- old-Python-2.5.cross/setup.py	2006-11-16 17:47:44.000000000 +0100
+++ new-Python-2.5.cross/setup.py	2006-11-16 17:47:44.000000000 +0100
@@ -17,6 +17,13 @@
 # This global variable is used to hold the list of modules to be disabled.
 disabled_module_list = []
 
+import os
+
+if os.environ.get('CROSS_COMPILING') == 'yes':
+    sysconfig.get_config_vars()
+    sysconfig._config_vars.update (os.environ)
+
+
 def add_dir_to_list(dirlist, dir):
     """Add the directory 'dir' to the list 'dirlist' (at the front) if
     1) 'dir' is not already in 'dirlist'

